systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -133,9 +117,6 @@
      * File paths for metrics YAMLs
      */
     String PATH_TO_KAFKA_METRICS_CONFIG = PATH_TO_PACKAGING_EXAMPLES + "/metrics/kafka-metrics.yaml";
-    String PATH_TO_KAFKA_CRUISE_CONTROL_METRICS_CONFIG = PATH_TO_PACKAGING_EXAMPLES + "/metrics/kafka-cruise-control-metrics.yaml";
-    String PATH_TO_KAFKA_CONNECT_METRICS_CONFIG = PATH_TO_PACKAGING_EXAMPLES + "/metrics/kafka-connect-metrics.yaml";
-    String PATH_TO_KAFKA_MIRROR_MAKER_2_METRICS_CONFIG = PATH_TO_PACKAGING_EXAMPLES + "/metrics/kafka-mirror-maker-2-metrics.yaml";
```

Comment:
KafkaConnectTemplates.PATH_TO_KAFKA_CONNECT_METRICS_CONFIG
KafkaMirrorMaker2Templates.PATH_TO_KAFKA_MIRROR_MAKER_2_METRICS_CONFIG
Maybe it would make sense to move it to constants instead?

Reply:
Good question. We should move all to Constants or move all to templates. @see-quick @im-konge wdyt?

Reply:
Moving to `Constants` make sense to me